gpt_index/readers/file/base.py

```diff
@@ -140,9 +146,10 @@ def load_data(self, concatenate: bool = False) -> List[Document]:
             List[Document]: A list of documents.
 
         """
-        data: Union[str, List[str]] = ""
+        data: Union[str, List[str], ImageParserOutput] = ""
```

Comment:
let's add a TODO to refactor later? this is kind of jank lol

Reply:
Definitely lol